lib/model_patches.rb

```diff
@@ -22,12 +34,21 @@ def self.top_requests
     end
 
     def law_used_full
-      _("access to information")
+      _("access to documents")
```

Comment:
I think here you want to change the `msgstr` in transifex rather than the `msgstr` e.g. https://github.com/mysociety/asktheeu-theme/blob/master/locale-theme/fr/app.po#L219-L220